pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -100,7 +100,9 @@ type FlannelNetworkingSpec struct {
 
 // CalicoNetworkingSpec declares that we want Calico networking
 type CalicoNetworkingSpec struct {
-	CrossSubnet bool `json:"crossSubnet,omitempty"` // Enables Calico's cross-subnet mode when set to true
+	// CPURequest CPU request of Calico container. Default: 100m
+	CPURequest  *resource.Quantity `json:"cpuRequest,omitempty"`
+	CrossSubnet bool               `json:"crossSubnet,omitempty"` // Enables Calico's cross-subnet mode when set to true
```

Comment:
I think this would look better like this
```suggestion
	// CrossSubnet enables Calico's cross-subnet mode when set to true 
	CrossSubnet bool `json:"crossSubnet,omitempty"` 
```